tests/python_tests/test_vlm_pipeline.py

```diff
@@ -93,6 +93,50 @@ def streamer(word: str) -> bool:
         assert res.texts[0] == ''.join(result_from_streamer)
 
 
+configs = [
+    get_greedy(),
+    get_beam_search(),
+]
+
+@pytest.mark.precommit
+@pytest.mark.nightly
+@pytest.mark.parametrize("config", configs)
+def test_vlm_continuous_batching(config, cache):
+    scheduler_config = SchedulerConfig()
+    models_path = get_ov_model(model_ids[0], cache)
+    ov_pipe = VLMPipeline(models_path, "CPU", scheduler_config=scheduler_config)
+    generation_config = config
+    generation_config.max_new_tokens = 30
+    image_links_list = [
+        [],
+        [image_links[0]]
+    ]
+
+    res_generate = []
+    for links in image_links_for_testing:
+        images = []
+        for link in links:
+            images.append(get_image_by_link(link))
+
+        res_generate.append(ov_pipe.generate(prompts[0], images=images, generation_config=generation_config))
+
+    cb_pipe = ContinuousBatchingPipeline(models_path, scheduler_config=scheduler_config, device="CPU")
+    tokenizer = cb_pipe.get_tokenizer()
+
+    for idx, links in enumerate(image_links_list):
+        images = []
+        for link in links:
+            images.append(get_image_by_link(link))
+        handle = cb_pipe.add_request(idx, prompts[0], images, generation_config)
+        while handle.get_status() != GenerationStatus.FINISHED:
+            cb_pipe.step()
+        outputs = handle.read_all()
+        for out_idx, output in enumerate(outputs):
+            text = tokenizer.decode(output.generated_ids)
+            assert text == res_generate[idx].texts[out_idx]
+            assert output.score == res_generate[idx].scores[out_idx]
```

Comment:
here we compare CB vs CB, which should be equal I hope.

Maybe we need to have 2 separate tests?
- ContinuousBatchingPipeline vs VLMPipeline default
- VLMPipeline with scheduler config vs VLMPipeline default